README.md

```diff
@@ -116,10 +116,15 @@ module.exports = function render(req, res, next) {
 
   const {statusCode, headers, body} = await parse(html, options);
   if (statusCode < 309 && statusCode > 300) {
-    return res.redirect(response.statusCode, headers.location);
+    return res.redirect(statusCode, location);
   }
 
-  res.status(statusCode || 200);
+  if (statusCode) {
```

Comment:
I'm not sure why this is needed, statusCode || 200 should be equivalent except if the statusCode is 0 which is not a valid response code anyways

Reply:
Ah, what a mistake. good eye ✅